static/app/components/replays/replayPlayer.tsx

```diff
@@ -20,6 +21,8 @@ function BasePlayerRoot({className}: Props) {
     fastForwardSpeed,
     isBuffering,
   } = useReplayContext();
+  const location = useLocation();
+  const isDetailsPage = location.pathname.includes('/replays/');
```

Comment:
I agree fully. 
If we added a prop, how many layers of components would we need to pass it through to get from the view to this component?

Reply:
> I agree fully. If we added a prop, how many layers of components would we need to pass it through to get from the view to this component?

I think it's about 3 levels. I'll go ahead and refactor it since we aren't happy with this approach